presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotColumnHandle.java

```diff
@@ -26,13 +27,18 @@
 public class TestPinotColumnHandle
 {
     private final PinotColumnHandle columnHandle = new PinotColumnHandle("columnName", VARCHAR, REGULAR);
+    private final PinotColumnHandle arrayColumnHandle = new PinotColumnHandle("arrayColumn", new ArrayType(VARCHAR), REGULAR);
 
     @Test
     public void testJsonRoundTrip()
     {
         String json = COLUMN_CODEC.toJson(columnHandle);
         PinotColumnHandle copy = COLUMN_CODEC.fromJson(json);
         assertEquals(copy, columnHandle);
```

Comment:
Should this be in a separate test ? Like testJsonRoundTripWithArrays ?

Reply:
done